web/app/ratings_helpers.py

```diff
@@ -0,0 +1,22 @@
+
+
+def rating_to_traditional_range(rating_f):
```

Comment:
the 'kyu-dan boundary' is between [-1,1] -- you can see how they're adjusted in the current regime here: https://github.com/usgo/AGA-Ratings-Program/blob/master/collection.cpp#L356.  

The math on that one is a little different; IIRC, it maybe enforces a lower bound of -30, but there is no upper bound enforced in the math.  The ratings that expand up past 7.0 are only anchored by the fact that there's a lot of priors, IIUC